components/combobox/combobox.jsx

```diff
@@ -532,6 +545,16 @@ class Combobox extends React.Component {
 		this.handleNavigateListboxMenu(event, { direction: 'next' });
 	};
 
+	handleKeyDownTab = () => {
+		if (this.selectedListboxRef) {
+			this.setState({
+				activeSelectedOption: this.props.selection[0],
```

Comment:
What's the reason for the tab reset here? We are not doing it in PillContainer right now either http://localhost:9001/?selectedKind=SLDSPillContainer&selectedStory=Base%20Pill%20Container&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Reply:
@interactivellama if you are focused on the combobox input, pressing tab should move focus to the listbox of pills. Before this code was added it would switch away from the entire combobox (and listbox) altogether.